emiliui.css

```diff
@@ -0,0 +1,31 @@
+h1{
+    background: #f5f5f5 ; 
+}
+body{
+    justify-content: center; 
+    padding: 20px;
+    background-color: rgba(251, 250, 252, 0.953);
+}
+.header{
```

Comment:
Header will likely be the same in all of the pages. We probably don't need a separate class for it.